src/main/webapp/partials/instructorHelpArchiving.jsp

```diff
@@ -0,0 +1,136 @@
+<%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8"%>
+<h4>
+    <a name="archiving">Archiving</a>
+</h4>
+<div id="contentHolder">
+    <br>
+    <ol style="list-style-type: none;">
+        <li>
+            <span class="text-bold">
+                <a name="archivingCourse">
+                    <h3>1. Archiving a course</h3>
+                </a>
+            </span>
+            <div>
+                You can archive a course so that it doesn't appear in your home page anymore. You cannot create feedback sessions, or enroll students, or edit an archived course.
```

Comment:
Second sentence: `You cannot edit, create feedback sessions or enroll students in an archived course.` (reason: `create feedback sessions` and `enroll students` need an `in`, but not `edit`).